### PR TITLE
Fixing bpm directory layout for automated import into vantiq namespace

### PR DESCRIPTION
**Pull request by pburma**
Tested with a new namespace via drag-n-drop and outcome was good.

### CODE REVIEW

File: Field Service Management/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Field Service Management
+This repository contains the Field Service Management blueprint and documentation. 
+
+To load from a VANTIQ Namespace locate the Projects --> Import button. Under Blueprints find Field Service Management and import the entire contents.
```
Comment (JakeSchmitz) [inline Field Service Management/README.md]: This content will be displayed after the user has already clicked Projects -> Import, and after they've selected Blueprints and selected "Field Service Management" as the blueprint to import, so I think this line can be skipped. The README should be more about what scenario the blueprint is for, and what components are included.

**Review by JakeSchmitz — CHANGES_REQUESTED** [review]
I think the README needs to be adjusted so it makes sense in the context of the import popup. Right now it reads more like a readme you would want to see in GitHub, which isn't the intended purpose. Otherwise this looks good to me.

File: Field Service Management/README.md (hunk 1 above)
Comment (pburma) [reply Field Service Management/README.md]: I updated the Readme per your comments.

**Review by JakeSchmitz — APPROVED** [review]
The recent changes look good to me. The readme may be a little too long now, but we'll have to wait for blueprints to be enabled in the UI to know for sure. 

The one last thing you should do is remove the VAIL files that are generated by apps and collaboration types. These will be regenerated when the collaboration type or app definition is imported. There are a couple reasons this is important: 

* Importing the VAIL files can race against the importing of the collaboration types (which is going to generate procedures and rules with the same names)
* If we find and fix bugs in code generation/ any of the activity pattern implementations those changes won't make it into the import of the blueprint because the generated VAIL in this repo will be out of date. 

I'm going to approve this now, so whenever you remove those files you should be able to merge this in.

Thanks for putting this together @pburma!

**Pull request merged**